rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -45,6 +46,8 @@ def featurize_training_examples(
 
     Returns:
         A list of attribute to features.
+        A dictionary of attribute that has sparse features to a dictionary of
+        a feature type to a list of different sparse feature sizes.

```

Comment:
You don't need to explain the data types of values returned. If you want to explain the schema of the return value, write that in the description of the docstring.